.changeset/new-islands-lick.md

```diff
@@ -0,0 +1,5 @@
+---
+'astro': minor
+---
+
+Added a property to image services to allow them to specify which properties of `getImage` / `Image` / `Picture` should be used for hashing the result files
```

Comment:
```suggestion
Adds a new property `propertiesToHash` to the Image Services API to allow specifying which properties of `getImage()` / `<Image />` / `<Picture />` should be used for hashing the result files.
```
Could be useful for some to have the property listed here! Also, please double check the wording still makes sense with the slight change!